cves/kernel/CVE-2020-14416.yml

```diff
@@ -454,7 +491,11 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    The main mistake here was failing to verify that the pointer is not null
+    before performing operations on it. Another failing to properly timing
```

Comment:
Second sentence is not a full sentence I think, and the answer could be expanded on based on the CWE mitigations

Reply:
Fixed wording and added a bit more detail on the mistake.